content/courses/native-onchain-development/program-derived-addresses.md

```diff
@@ -294,50 +389,55 @@ fn get_associated_token_address_and_bump_seed_internal(
 }
 ```
 
-The mappings between seeds and PDA accounts that you use will be highly
-dependent on your specific program. While this isn't a lesson on system design
-or architecture, it's worth calling out a few guidelines:
+The mappings between seeds and PDA accounts you use will depend highly on your
+specific program. While this isn't a lesson on system design or architecture,
+it's worth calling out a few guidelines:
 
-- Use seeds that will be known at the time of PDA derivation
-- Be thoughtful about what data is grouped together into a single account
+- Use seeds known at the time of PDA derivation
+- Be thoughtful about how you group data into a single account
 - Be thoughtful about the data structure used within each account
 - Simpler is usually better
 
 ## Lab
 
-Let’s practice together with the Movie Review program we've worked on in
-previous lessons. No worries if you’re just jumping into this lesson without
-having done the previous lesson - it should be possible to follow along either
-way.
+Let's practice with the Movie Review program we've worked on in previous
+lessons. No worries if you're jumping into this lesson without doing the last
+lesson - it should be possible to follow along either way.
 
 As a refresher, the Movie Review program lets users create movie reviews. These
-reviews are stored in an account using a PDA derived with the initializer’s
-public key and the title of the movie they are reviewing.
+reviews are stored in an account using a PDA derived from the initializer's
+public key and the movie title they are reviewing.
 
-Previously, we finished implementing the ability to update a movie review in a
-secure manner. In this lab, we'll add the ability for users to comment on a
-movie review. We'll use building this feature as an opportunity to work through
-how to structure the comment storage using PDA accounts.
+Previously, we finished implementing the ability to update a movie review
+securely. In this lab, we'll add the ability for users to comment on a movie
+review. We'll use building this feature as an opportunity to work through how to
+structure the comment storage using PDA accounts.
 
-#### 1. Get the starter code
+### 1. Get the starter code
 
 To begin, you can find
 [the movie program starter code](https://github.com/Unboxed-Software/solana-movie-program/tree/starter)
```

Comment:
This repo needs to be updated. https://github.com/Unboxed-Software/solana-movie-program - send PRs to that repo and then I'll marge it and move it to solana-developers

Reply:
I send a PR a few weeks ago - https://github.com/Unboxed-Software/solana-movie-program/pull/5

Waiting for merge so that I can update the link in the article

Reply:
I have also updated the solution and main branch today

https://github.com/Unboxed-Software/solana-movie-program/pull/6
https://github.com/Unboxed-Software/solana-movie-program/pull/7

Reply:
I have updated the frontend solution for this course
https://github.com/Unboxed-Software/solana-movie-frontend/pull/16

Reply:
Updated the student-intro-program
https://github.com/448-OG/solana-student-intro-program

This has 3 branch changes but all of them upgrading the dependencies and replacing deprecated borsh import. Can the repo just be cloned or should I create 3 PRs for the same thing?

Reply:
Your student-intro-program repo has been cloned to https://github.com/solana-developers/student-intro-program

Reply:
> Sorry about the wait BTW, we had in person NYC training then RustConf then Breakpoint.

No problem, guessed you were at Breakpoint. Watched it online you guys are doing awesome work.